apps/web/.env.example

```
@@ -1,27 +1,37 @@
+# Test/Development Environment (currently active)
+NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY=
+CLERK_SECRET_KEY=
 
-# Authentication
-# @thedaviddias/auth
-GITHUB_TOKEN=
+# -----------------------------------------------------------------------------
+# 🌐 APPLICATION CONFIGURATION
+# -----------------------------------------------------------------------------
+# Public URL for the web application
+NEXT_PUBLIC_WEB_URL="https://localhost:3000"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)